client/src/components/MarketUpdate/UpdateTurnipSelectors.js

```diff
@@ -0,0 +1,24 @@
+export function createTurnipPayload(enteredTurnipData, islandId) {
+	const { date, timeOfDay, price } = enteredTurnipData;
+	const year = date.getFullYear();
+	const month = date.getMonth();
+	const day = date.getDate();
+	const time = timeOfDay === "am" ? 1 : 13;
+	const parsedDate = new Date(year, month, day, time);
+	return {
+		date: parsedDate,
+		price: parseInt(price),
+		islandId: islandId,
+	};
+}
+
+export function findIslandByIslandId(islands = [], islandId) {
+	let islandIndex = 0;
+	for (let island of islands) {
+		if (island._id === islandId) {
+			return islandIndex;
+		}
+		islandIndex++;
+	}
+	return undefined;
```

Comment:
If you don't return anything, doesn't it return "undefined"? Or are you writing it to be verbose?

Reply:
Haha mostly being verbose, got used to TypeScript complaining when there's no return value